Classes/Importer/AbstractImporter.php

```diff
@@ -551,27 +601,46 @@ protected function getLabelFromRecordData(array $data)
      *
      * The storage node is either created or just retrieved and finally stored in $this->storageNode.
      *
-     * @param string $nodePath Absolute or relative (to the site node) node path of the storage node
+     * @param string $nodePathOrIdentifier Absolute or relative (to the site node) node path of the storage node
```

Comment:
`A nodeIdentifier or an absolute or relative (to the site node) node path of the storage node`